src/Controller/EventController.php

```diff
@@ -42,6 +46,118 @@ public function viewEvents(Request $request, EntityManagerInterface $manager, Pa
     }
 
     /**
+     * @Route("/events/create", name="event_create")
+     * @Security("is_granted('ROLE_ADMIN')")
+     */
+    public function create(Request $request)
+    {
+        $event = new Event();
+
+        $form = $this->createForm(EventCreateType::class, [
+            'action' => $this->generateUrl('event_create')
+        ]);
+
+        $form->handleRequest($request);
+
+        if($form->isSubmitted() && $form->isValid())
+        {
+            $file = $request->files->get('event_create')['photo'];
+            if($file->getClientSize() > 3000000) {
+                $this->addFlash('danger', 'Failo dydis yra per didelis.');
+            }
+            if($file->getClientMimeType() === 'image/png' || $file->getClientMimeType() === 'image/jpeg') {
+                $uploads_directory = $this->getParameter('events_directory');
+                $filename = md5(uniqid()) . '.' . $file->guessExtension();
+
+                $file->move(
+                    $uploads_directory,
+                    $filename
+                );
+
+                $event->setTitle($form->get('title')->getData());
```

Comment:
All of this would not be needed if you passed $event to the form. If you choose not to, $event=$form->getData() would work too, but still, the first option is better

Reply:
Fixed this in delete event branch.